docs/sources/tempo/configuration/sidecar-proxy.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Configure sidecar proxies
+menuTitle: Configure sidecar proxies
+description: Configure Tempo Distributed to run with sidecar proxies
+weight: 36
+---
+
+# Run Tempo Distributed with sidecar proxies
```

Comment:
This section is getting the reader up to speed on how Tempo works by default (before we start changing things) so maybe it should have a subheader like `## Background` or `## How Tempo Pods Communicate` ?

Reply:
Done. Thank you for the suggestion.